radiomics/imageoperations.py

```diff
@@ -13,42 +13,64 @@
 logger = logging.getLogger(__name__)
 
 
-def getHistogram(binwidth, parameterValues):
-  """
+def getBinEdges(binwidth, parameterValues):
```

Comment:
I just now realized that numpy has its own set of strategies for calculating bins. I think it would be helpful to give user an option to access those numpy strategies. I don't think this is possible right now.

Related question - should we raise the question of bin calculation withe radiomics standardization effort group? It does not look like their document discusses this at the moment.